website/docs/components/form/checkbox/partials/code/component-api.md

```diff
@@ -1,95 +1,62 @@
 ## Component API 
 
-The `Form::Checkbox` component has three different variants, with their own APIs:
+The Checkbox component has three different variants with their own APIs:
 
-- `Form::Checkbox::Group` - the "group" parent component: a `<legend>` (optional), a list of fields, and error messaging
-- `Form::Checkbox::Field` - the "field" parent component: the `<input>` control, with label, helper text and error messaging (in a wrapping container)
-- `Form::Checkbox::Base` - the "basic" component: just the `<input>` control
+- `Form::Checkbox::Base` - the base component: the `<input>` control
+- `Form::Checkbox::Field` - the field component: the `<input>` control, with label, helper text, and error messaging (in a wrapping container)
+- `Form::Checkbox::Group` - the group component: a `<legend>` (optional), a list of fields, and error messaging
```

Comment:
Since this is the parent component, it should be first in the list (consistent with the expectations of API docs)